src/renderer/components/media/MediaControls.vue

```diff
@@ -2,18 +2,18 @@
 <template>
   <v-row>
     <v-app-bar fixed>
-      <v-col class="text-left" cols="auto">
+      <v-col class="text-left" cols="4">
         <v-btn icon aria-label="Toggle prefix" @click="togglePrefix()">
           <font-awesome-icon :icon="faEye" />
           <font-awesome-icon :icon="faListOl" />
         </v-btn>
       </v-col>
       <v-col class="text-center d-flex justify-center">
-        <v-btn color="secondary" :disabled="mediaActive" @click="clearDate()">
+        <v-btn class="px-3" color="secondary" :disabled="mediaActive" @click="clearDate()">
           {{ date }}
         </v-btn>
       </v-col>
-      <v-col class="text-right" cols="auto">
+      <v-col class="text-right" cols="4">
```

Comment:
Although I agree that centered looks better, it doesn't work when you enable PowerPoint shortcuts and are on the lowest width.

Reply:
Either we have to move some of those control buttons, increase the min width or do some tweaks to above values.

Reply:
![Screenshot from 2022-09-27 18-23-49](https://user-images.githubusercontent.com/46671786/192581745-d7a90a2f-7984-4222-ac67-309b7d605a45.png)

Reply:
What then if we moved the folder name to the left, and have the other icons on the right?

Reply:
How do you feel about this?
![Screenshot from 2022-09-27 18-49-38](https://user-images.githubusercontent.com/46671786/192589456-4c45f427-b9b1-44e3-9897-fc95c08cf043.png)

Reply:
Nice. I like it

Reply:
I pushed it to your branch, so you'll need to do a `git pull`